docs/_posts/murat-gunay/2022-02-23-bert_sequence_classifier_trec_coarse_pipeline_en.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: model
+title: Classifier Pipeline to Identify Trec Data Classes
```

Comment:
Pipeline to Classify Texts into TREC-6 Classes

Reply:
fixed